src/Pinch/Internal/Value.hs

```diff
@@ -70,6 +70,7 @@ data Value a where
 
     VMap  :: forall k v. (IsTType k, IsTType v)
           => !(FoldList (MapItem k v)) -> Value TMap
+    VNullMap :: Value TMap
```

Comment:
You'll need to handle `VNullMap` in the `Pinchable` instances for maps. Currently they only expect a `VMap`.